api/src/services/configuration-service.ts

```diff
@@ -39,7 +37,7 @@ export class ConfigurationService {
 
 	config: Config = {
 		port: process.env.PORT ? Number.parseInt(process.env.PORT, 10) : 3000,
-		apiVersion: process.env.API_VERSION,
+		apiVersion: 'v0.1',
```

Comment:
This value is only read in the index.ts file: it should go there (close to /api)
Please remove the apiVersion field from Config and anywhere is used.

Reply:
It is also used on /info route and in swagger doc, I thought it would be better to have all configurable variables in one place so you know where to adjust it. wdyt?

Reply:
Ok I have no problem in putting this number in a single place: only not in Config because is not a parameter.